### PR TITLE
Feature/travis

### PR DESCRIPTION
**Pull request by ozzyian**
Added support for travis
Two simple unit tests for utils.py

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.7"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script:
+  - pytest
+before_script: cd src
```
Comment (LucasGronlund) [inline .travis.yml:10]: files should end with new line

File: src/test/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    parsed_users = utils.parse_users(users)
+    assert parsed_users == {"Lucas": "lucas123"}
+
+
+def test_get_reports_key():
+    guild = "Inheritance"
+    server = "Razorgore"
+    region = "EU"
+    expected = f"<{guild}>-<{server}>-<{region}>"
+    assert utils.get_reports_key(guild, server, region) == expected
```
Comment (LucasGronlund) [inline src/test/test_utils.py:15]: newline

Comment (ozzyian) [reply src/test/test_utils.py:15]: There is a newline :(

Comment (LucasGronlund) [reply src/test/test_utils.py:15]: my bad sir!

**Pull request merged**